### PR TITLE
added warning about minio

### PR DESCRIPTION
**Pull request by sandstromviktor**
## Description

Added warning text about minio for release.

### CODE REVIEW

File: templates/apps/create.html [hunk 1]
```diff
@@ -39,13 +39,20 @@ <h1 class="h3 mb-3">Create {{ app.name }}</h1>
...
 
+
                     <div class="p-3 mb-3">
-                        <h4>Important information</h4>
-                    <p>You are about to activate file manager on SciLifeLab Serve. You can use it to upload or download files to a volume associated with this project. This service will be active for 24 hours and automatically terminated afterwards. The uploaded files will stay on the volume even after this service has been terminated. All credentials needed to use the service will be provided after activation.</p>
-                    <p>Click 'Activate' to activate file manager</p>
 
+                        <div role="alert" class="alert alert-warning text-center">
+                          <h4><i class="bi bi-exclamation-triangle px-2"></i>Important information</h4>
+                          This is an experimental functionality.<br>
```
Comment (alfredeen) [inline templates/apps/create.html:47]:
```suggestion
                          This is an experimental feature.<br>
```

**Review by alfredeen — CHANGES_REQUESTED** [review]

File: templates/apps/create.html [hunk 2]
```diff
@@ -39,13 +39,20 @@ <h1 class="h3 mb-3">Create {{ app.name }}</h1>
...
+
                     <div class="p-3 mb-3">
-                        <h4>Important information</h4>
-                    <p>You are about to activate file manager on SciLifeLab Serve. You can use it to upload or download files to a volume associated with this project. This service will be active for 24 hours and automatically terminated afterwards. The uploaded files will stay on the volume even after this service has been terminated. All credentials needed to use the service will be provided after activation.</p>
-                    <p>Click 'Activate' to activate file manager</p>
 
+                        <div role="alert" class="alert alert-warning text-center">
+                          <h4><i class="bi bi-exclamation-triangle px-2"></i>Important information</h4>
+                          This is an experimental feature.<br>
+                          If you experience issues, please contact us as serve@scilifelab.se
```
Comment (churnikov) [inline templates/apps/create.html:48]:
```suggestion
                          If you experience issues, please contact us at serve@scilifelab.se
```

**Review by churnikov — APPROVED** [review]
Other than one typo I'm okay

**Review by alfredeen — APPROVED** [review]
lgtm

**Pull request merged**